config.global.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 data:
-  # Change redirect code to 301 as 308 is not understood by social media bots
   body-size: "0"
-  http-redirect-code: "301"
+  # Change redirect code to 301 as 308 is not understood by social media bots
+  http-redirect-code: 301
```

Comment:
@tbille trying to QA locally, I now get:

``` bash
$ ./qa-deploy --production old-docs.jujucharms.com --tag latest
configmap/global-config unchanged
Error from server (BadRequest): error when creating "STDIN": ConfigMap in version "v1" cannot be handled as a ConfigMap: v1.ConfigMap.Data: ReadString: expects " or n, but found 3, error found in #10 byte of ...|ct-code":301,"map-ha|..., bigger context ...|v1","data":{"body-size":"0","http-redirect-code":301,"map-hash-bucket-size":"128","proxy-body-size":|...
```

The fix is to make this value a string. Why did you have to make it an int? How come it doesn't work in my microk8s?
```